dot/state/offline_pruner.go

```diff
@@ -160,54 +171,35 @@ func (p *OfflinePruner) Prune() error {
 		}
 	}()
 
-	prunedDB, err := utils.LoadBadgerDB(p.prunedDBPath)
-	if err != nil {
-		return fmt.Errorf("failed to load DB %w", err)
-	}
-	defer func() {
-		closeErr := prunedDB.Close()
-		switch {
-		case closeErr == nil:
-			return
-		case err == nil:
-			err = fmt.Errorf("cannot close pruned database: %w", closeErr)
-		default:
-			logger.Errorf("cannot close pruned database: %s", err)
-		}
-	}()
-
-	writer := prunedDB.NewStreamWriter()
-	if err = writer.Prepare(); err != nil {
-		return fmt.Errorf("cannot create stream writer in out DB at %s error %w", p.prunedDBPath, err)
-	}
-
-	// Stream contents of DB to the output DB.
 	stream := inputDB.NewStream()
-	stream.LogPrefix = fmt.Sprintf("Streaming DB to new DB at %s ", p.prunedDBPath)
-
 	stream.ChooseKey = func(item *badger.Item) bool {
 		key := string(item.Key())
```

Comment:
wondering if it is a good use case for using the `[]byte <> string` conversion w/o allocation: https://go.dev/play/p/P__twwbKG0O

Reply:
NO 🤣